drivers/soundwire/intel_ace2x.c

```diff
@@ -15,6 +15,22 @@
 #include "bus.h"
 #include "intel.h"
 
+/*
+ * shim_vs ops
+ */
+
+static void intel_shim_vs_init(struct sdw_intel *sdw)
```

Comment:
First I read this as "shim **VS** init" ;)
What is 'shim_vs'?

Reply:
shim vendor-specific. I'll clarify, if it's confusing then someone else will also have the same issue.

Reply:
done